fml/message_loop.h

```diff
@@ -27,7 +27,6 @@ class MessageLoopImpl;
 /// \see fml::Wakeable
 class MessageLoop {
  public:
-  FML_EMBEDDER_ONLY
```

Comment:
@chinmaygarde not sure what to do here. This macro is flagging usages of this API in flutter_main.cc, it looks like those hadn't been touched in a while. Is there an alternatie usage of this API I can switch to?

Reply:
So the `GetCurrent` cannot be supported in all configurations because the thread or embedder may not have the ability to fetch the thread local task runner and instead must already have a handle to its task runner. So this is warning (via deprecation tags) that the API may only be used on embedder that have known safe access to the TLS message loop.

If the embedder knows that it is safe to access the TLS message loop slot, it can `#define FML_USED_ON_EMBEDDER` to make this available. `flutter_main.cc` should already have this and you shouldn't have to mess with this here.

Ideally, we should get rid of GetCurrent altogether but that is a migration.